src/Device/DeviceSubTypeEnum.cs

```diff
@@ -115,6 +115,12 @@ public enum DeviceSubType
         HLA_VIRT, ///< Виртуальная сигнальная колонна (без привязки к модулям).
         HLA_IOLINK, ///< Сигнальная колонна IO-Link (настраиваемая)
 
+        //G
+        G = 1, ///< IO-Link блок питания с автоматическим выключателем.
+        G_IOL_4, ///< Phoenix Contact 4 канала.
+        G_IOL_8, ///< Phoenix Contact 8 каналов.
+        G_VIRT, ///< Виртуальный блок питания с автоматическим выключателем (без привязки к модулям).
```

Comment:
Данные подтипы не нужны
```suggestion
        G_IOL_4, ///< Phoenix Contact 4 канала.
        G_IOL_8, ///< Phoenix Contact 8 каналов.
```